pkg/cqldb/hostDialer.go

```diff
@@ -0,0 +1,100 @@
+package cqldb
```

Comment:
This file has 0% code coverage, can we add some tests?

Reply:
Fixed